scripts/dev

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env bash
 
-go run main.go start
+set -e -x
```

Comment:
I wonder what's this?

Reply:
```
-e  Exit immediately if a command exits with a non-zero status.
-x  Print commands and their arguments as they are executed
```

`-e` is useful for multiple commands in shell scripts